assets/src/components/customers/CustomersTableContainer.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {Box, Flex} from 'theme-ui';
+import {Button} from '../common';
+import * as API from '../../api';
+import logger from '../../logger';
+import {Customer, Pagination} from '../../types';
+import CustomersTable from './CustomersTable';
+
+type Props = {
+  currentlyOnline?: any;
+  defaultFilters?: Record<string, any>;
+  actions?: (
+    refreshCustomerData: (filters?: any) => void
+  ) => React.ReactElement;
+};
+type State = {
+  loading: boolean;
+  query: string;
+  customers: Array<Customer>;
+  filteredCustomers: Array<Customer>;
+  pagination: Pagination;
+};
+
+const DEFAULT_PAGE_SIZE = 10;
+
+class CustomersTableContainer extends React.Component<Props, State> {
```

Comment:
Thanks for separating to another component!

Reply:
no prob, just wanted to remove a bunch of the duplicate code :) :+1: